src/content/docs/solvro/Strategy.mdx

```diff
@@ -0,0 +1,164 @@
+---
+title: Strategia KN Solvro
+sidebar:
+  order: 1
+description: Długoterminowy plan strategiczny KN Solvro
+---
+
+import {
+  Steps,
+  Card,
+  CardGrid,
+  LinkCard,
+  Aside,
+} from "@astrojs/starlight/components";
+
+> ### <i>"Otwieramy drzwi do świata IT" ~ motto KN Solvro</i>
```

Comment:
co to za heading, nie widzę żeby on cokolwiek opisywał 

użyj pogrubienia zwykłego jeśli zależy Ci bardzo by to wstawić

Reply:
Zmieniłem na pogrubienie, ale głównie chodziło, żeby tekst  był większy i wyraźnie wyróżniony